pkg/graveler/graveler.go

```diff
@@ -1557,9 +1556,18 @@ func (g *KVGraveler) listStagingArea(ctx context.Context, b *Branch) (ValueItera
 		return it, nil
 	}
 
-	itrs := []ValueIterator{it}
+	itrs, err := g.listSealedTokens(ctx, b)
+	if err != nil {
+		return nil, err
+	}
+	itrs = append(itrs, it)
```

Comment:
`it` should be the first iterator. If our tests didn't catch this, it's very bad 😭

Reply:
done